lib/Phactory/Utils.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Phactory;
+
+class Utils {
+  public static function getValueOrReturn($value) {
+    if(is_object($value) && is_callable($value)
```

Comment:
It is because, if I pass the name of a know function (`'count'`, for example), `is_callable` would return true, but there's no way to know if user wanted the function `count` or the string `'count'`.